site/src/pages/WorkspacePage/hooks.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  getWorkspaceBuildByNumber,
+  stopWorkspace,
+  startWorkspace,
+} from "api/api"
+import { delay } from "utils/delay"
+import { ProvisionerJob, WorkspaceBuild } from "api/typesGenerated"
+import { useMutation } from "@tanstack/react-query"
+
+export function waitForBuild(build: WorkspaceBuild) {
```

Comment:
Why do we need to export this function? Also, this is a really good function to be on `api.ts` very useful for other parts of the code we have.